app/charts/line/chart-lines.tsx

```diff
@@ -37,6 +37,7 @@ const ChartLines = memo((props: ChartProps<LineConfig>) => {
         <ChartSvg>
           <AxisHeightLinear /> <AxisTime /> <AxisTimeDomain />
           <Lines />
+          <Points chartConfig={chartConfig} />
```

Comment:
@squiles I wonder if it wouldn't be a bit better to have a conditional here, like:

```tsx
{chartConfig.fields.y.showDots ? <Points dotsSize={chart.fields.y.showDotsSize} /> : null}
```

so that we don't need to compute `Points`-related things (like e.g. `ticksPos` in `useMemo`) when the dots aren't supposed to be shown?

Then you could also remove `undefined` from the switch in `getPointRadius`, as when it's undefined, technically we shouldn't show any dots, right?

Reply:
yeah, definitely makes sense. I've changed it accordingly 👍